src/seccomp/src/bin.rs

```diff
@@ -0,0 +1,567 @@
+        assert_eq!(format!("{}", Error::MissingInputFile), "Missing input file");
+    }
+    #[test]
+    fn test_get_argument_values() {
+        let arg_parser = build_arg_parser();
+        let default_out_file_name = "bpf.out";
+        // correct arguments
+        let arguments = &mut arg_parser.arguments().clone();
+        arguments
+            .parse(
+                vec![
+                    "seccompiler",
+                    "--input_file",
+                    "foo.txt",
+                    "--target_arch",
+                    "x86_64",
+                ]
+                .into_iter()
+                .map(String::from)
+                .collect::<Vec<String>>()
+                .as_ref(),
+            )
+            .unwrap();
+        assert_eq!(
+            get_argument_values(arguments).unwrap(),
+            Arguments {
+                input_file: "foo.txt".to_string(),
+                output_file: default_out_file_name.to_string(),
+                target_arch: "x86_64".to_string()
+            }
+        );
+
+        let arguments = &mut arg_parser.arguments().clone();
+        arguments
+            .parse(
+                vec![
+                    "seccompiler",
+                    "--input_file",
+                    "foo.txt",
+                    "--target_arch",
+                    "x86_64",
+                    "--output_file",
+                    "/path.to/file.txt",
+                ]
+                .into_iter()
+                .map(String::from)
+                .collect::<Vec<String>>()
+                .as_ref(),
+            )
+            .unwrap();
+        assert_eq!(
+            get_argument_values(arguments).unwrap(),
+            Arguments {
+                input_file: "foo.txt".to_string(),
+                output_file: "/path.to/file.txt".to_string(),
+                target_arch: "x86_64".to_string()
+            }
+        );
+
+        // no args
+        let arguments = &mut arg_parser.arguments().clone();
+        assert!(arguments
+            .parse(
+                vec!["seccompiler"]
+                    .into_iter()
+                    .map(String::from)
+                    .collect::<Vec<String>>()
+                    .as_ref(),
+            )
+            .is_err());
+
+        // invalid --target_arch
+        let arguments = &mut arg_parser.arguments().clone();
+        arguments
+            .parse(
+                vec![
+                    "seccompiler",
+                    "--input_file",
+                    "foo.txt",
+                    "--target_arch",
+                    "x86_64das",
+                    "--output_file",
+                    "/path.to/file.txt",
+                ]
+                .into_iter()
+                .map(String::from)
+                .collect::<Vec<String>>()
+                .as_ref(),
+            )
+            .unwrap();
+        assert!(get_argument_values(arguments).is_err());
+        // invalid --target_arch
```

Comment:
You have a duplicate case of invalid `--target-arch` here.

Reply:
will remove it